docs/REST API/rest-api.md

```diff
@@ -9183,20 +9149,32 @@ Returns the list of App Store (VPP) apps purchased in Apple Business Manager. Ap
 
 ```json
 {
-  "app_store_apps": {
+  "app_store_apps": [
     {
       "name": "Xcode",
       "icon_url": "https://is1-ssl.mzstatic.com/image/thumb/Purple211/v4/f1/65/1e/a4844ccd-486d-455f-bb31-67336fe46b14/AppIcon-1x_U007emarketing-0-7-0-85-220-0.png/512x512bb.jpg",
       "latest_version": "15.4",
-      "app_store_id": "497799835"
+      "app_store_id": "497799835",
+      "added": true,
```

Comment:
@marko-lisica FYI it looks like we added this `added` boolean. 

@getvictor do you know why we added that? The plan was for the backend to filter the list and only return apps that haven't been added to the team yet. That was the user doesn't have to think about the ones already added.

I think this solution is ok too. We have to go with it! Just want to understand if this was easier to build or we just miscommunicated that design decision.

Reply:
@noahtalerman Actually, `added` is not there -- the backend returns apps that haven't been added to the team yet. So, you can remove it from API reference.